### PR TITLE
Fix some issues with metastation

### PR DESCRIPTION
**Pull request by Couls**
**What does this PR do:**
- Replace chemistry and scichem lockers with reagent lockers
- add in IV bags and replaced the blood bags in medical
- add shutter button to armory
- move the fridge from the corner between a chemical heater so that it can be unwrenched and repurposed
- give robotics ipc repair kits and anathestic tanks
- add lethal ammo boxes for the shotgun in armory(slug and buckshot)

**Images of sprite/map changes (IF APPLICABLE):**
Before:
image
image
image
image
After:
image
image
image
image

**Changelog:**
:cl:
add:Replace chemistry and scichem lockers with reagent lockers[Metastation]
add:IV bags and replaced the blood bags in medical[Metastation]
add:shutter button to armory[Metastation]
tweak:move the fridge from the corner between a chemical heater so that it can be unwrenched and repurposed[Metastation]
tweak:give robotics ipc repair kits and anathestic tanks[Metastation]
add: lethal ammo boxes for the shotgun in armory(slug and buckshot)[Metastation]
add: fuel tank dispensers for scichem, air alarm in medchem[Metastation]
add: a roboticist locker into the robotics room[Metastation]
/:cl:

### CODE REVIEW

**Comment by farie82** [comment]
CL should specify that it's metastation. But nice job

**Comment by craftxbox** [comment]
would be good to add more large beakers to scichem

**Comment by SkeletalElite** [comment]
Chaplain gets two null rods and two soul stones on meta as well. May as well fix that while you're at it. (He has a soul stone in locker and null rod in hand and also soul stone+ null rod in confession booth resulting in him having two)

**Comment by KasparoVy** [comment]
@SkeletalElite Holy shit, Chaplain with two ling arms

**Comment by Couls** [comment]
> would be good to add more large beakers to scichem, and a fuel tank

Scichem has fuel tank dispensers in the corner and I feel like enough large beakers to last until an autolathe or bluespace beakers are researched

**Comment by KasparoVy** [comment]
I've seen the double ling arms. I've seen the other side.

File: paradise.dme [hunk 1]
```diff
@@ -11,7 +11,7 @@
 // END_PREFERENCES
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
-#include "_maps\cyberiad.dm"
```
Comment (variableundefined) [inline paradise.dme]: Urm.

This is gonna switch the map

Comment (Couls) [reply paradise.dme]: Oh whoops I’ll fix that with a few other issues with metastation once I have some time to work on this

**Comment by Couls** [comment]
Going to close this for now and reopen it when I can dedicate some time

**Pull request closed without merging**